ios-10-3-dep-8-0-bitcode.cmake

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2016, Ruslan Baratov
+# All rights reserved.
+
+if(DEFINED POLLY_IOS_10_3_DEP_8_0_BITCODE_CMAKE_)
+  return()
+else()
+  set(POLLY_IOS_10_3_DEP_8_0_BITCODE_CMAKE_ 1)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_clear_environment_variables.cmake")
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_init.cmake")
+
+set(IOS_SDK_VERSION 10.3)
+set(IOS_DEPLOYMENT_SDK_VERSION 8.0)
+
+set(POLLY_XCODE_COMPILER "clang")
+polly_init(
+    "iOS ${IOS_SDK_VERSION} / Deployment ${IOS_DEPLOYMENT_SDK_VERSION} / Universal (iphoneos + iphonesimulator) / \
+${POLLY_XCODE_COMPILER} / \
+bitcode / \
+c++11 support"
+    "Xcode"
+)
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_common.cmake")
+
+include(polly_fatal_error)
+
+# Fix try_compile
+set(MACOSX_BUNDLE_GUI_IDENTIFIER com.example)
+set(CMAKE_MACOSX_BUNDLE YES)
+set(CMAKE_XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY "iPhone Developer")
+
+set(IPHONEOS_ARCHS armv7;armv7s;arm64)
+set(IPHONESIMULATOR_ARCHS i386;x86_64)
+
+include("${CMAKE_CURRENT_LIST_DIR}/compiler/xcode.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/os/iphone.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/flags/cxx11.cmake")
+
+set(CMAKE_XCODE_ATTRIBUTE_ENABLE_BITCODE YES)

```

Comment:
This flag is not "visible" by toolchain-id calculation module in Hunter (there is no C++ macro to detect this feature). In this case we have to mark toolchain explicitly. Please move this code to separate module `flags/bitcode.cmake` and add `list(APPEND HUNTER_TOOLCHAIN_UNDETECTABLE_ID "bitcode")`.

Example:
* https://github.com/ruslo/polly/blob/8d8b688ea809743a7bca80e34822f9395fbcd211/flags/static.cmake#L15-L17